sdk/Lusid.Sdk.Tests/tutorials/Ibor/GenericReconciliation.cs

```diff
@@ -0,0 +1,773 @@
+                quotePrice, traderName);
+
+            // create the reconciliation request
+            var reconciliation = new ReconciliationRequest(valuationRequestOne, valuationRequestTwo);
+            var reconciliationResponse = _apiFactory.Api<ReconciliationsApi>().ReconcileGeneric(reconciliation);
+
+            // Get the reconciliation of our equity. 
+            var equityComparison = reconciliationResponse.Comparisons.Single();
+
+            // The trader name properties on the two portfolios have different names and both have been compared upon. 
+            Assert.That(equityComparison.ResultComparison.ContainsKey($"Transaction/{_portfolioOneScope}/TraderName"));
+            Assert.That(equityComparison.ResultComparison.ContainsKey($"Transaction/{_portfolioTwoScope}/TraderName"));
+
+            // Assert that the reconciliation failed on comparing the properties.
+            Assert.That(equityComparison.ResultComparison[$"Transaction/{_portfolioOneScope}/TraderName"].ToString()
+                .Contains("Failed"));
+            Assert.That(equityComparison.ResultComparison[$"Transaction/{_portfolioTwoScope}/TraderName"].ToString()
+                .Contains("Failed"));
+
+            // As we have a property which has a portfolio dependant name, need to tell the portfolio that this is the case in order to map them together.
+            var mapping = new ReconciliationLeftRightAddressKeyPair($"Transaction/{_portfolioOneScope}/TraderName",
+                $"Transaction/{_portfolioTwoScope}/TraderName");
+
+            // create the new reconciliation request with the mapping 
+            reconciliation = new ReconciliationRequest(valuationRequestOne, valuationRequestTwo,
+                new List<ReconciliationLeftRightAddressKeyPair>() {mapping});
+            reconciliationResponse = _apiFactory.Api<ReconciliationsApi>().ReconcileGeneric(reconciliation);
+
+            equityComparison = reconciliationResponse.Comparisons.Single();
+
+            // Check that all the comparisons return an exact match
+            Assert.That(equityComparison.ResultComparison.All(x => x.Value.ToString().Equals("ExactMatch")));
+
+            // Check that the property strings where correctly matched.
+            // The difference and comparison dictionaries retain the name of the Right valuation result.
+            Assert.That(equityComparison.Left.ContainsKey($"Transaction/{_portfolioOneScope}/TraderName"));
+            Assert.That(equityComparison.Right.ContainsKey($"Transaction/{_portfolioTwoScope}/TraderName"));
+            Assert.That(equityComparison.Difference.ContainsKey($"Transaction/{_portfolioTwoScope}/TraderName"));
+            Assert.That(!equityComparison.Difference.ContainsKey($"Transaction/{_portfolioOneScope}/TraderName"));
+        }
+
+        /// <summary>
+        /// The default behaviour for comparing numeric values is for exact tolerance. This means only exactly matching
```

Comment:
A little bit terse. Given this is to explain to a user how the comparison is working, I'd copy or write a bit more of a generic explanation even at the risk of duplication.

Reply:
modified to provide more detail